### ISSUE TITLE
podcast:contentRating

### ISSUE THREAD

**Issue by jmikedupont2** [Podcastindex-org/podcast-namespace#151]
There is an ontology that already specifies the content rating that we can use from the wwwc

see this document here:  https://www.w3.org/TR/mediaont-10/
and  https://www.w3.org/2008/WebVideo/Annotations/drafts/ontology10/WD/CableLabs1.html
Example 
```
<app_data app=""MOD"" name=""MSORating"" value=""Age-14"/" > ~~~</app_data>
<app_data app="”MOD”" name="”Rating”" value="”TV-G”/"> ~~~</app_data>
```

**Comment by daveajones** [comment]
Thanks for this. Do you have any idea if this got ratified to a standard at any point?

**Comment by jmikedupont2** [comment]
It is just a recommendation from 2012 but I think we could at least align
ourselves in some way with it or at least look how it and other ontology
overlap with what we're doing

**Comment by jmikedupont2** [comment]
I found another example of this same verb in schema.org, that is a "standard" of some type. https://schema.org/contentRating

**Comment by jmikedupont2** [comment]
So my recommendation is to allow multiple elements of this to be attached to an episode, a feed or even a section of transcript or clip and to add in the attribute of the system used, there are 28 different ones here on wikipedia https://en.wikipedia.org/wiki/Category:Motion_picture_rating_systems.

Here is another good overview of rating
https://nsindex.net/wiki/Motion_picture_rating_system

**Comment by theDanielJLewis** [comment]
(See #24 and #2 for more context.)

I suggest staying away from age advisories because they're almost meaningless and they don't help adults with conservative preferences.

That's why I like something that more indicates _what_ is in the podcast/episode, and then let the listener or their parent decide on its appropriateness. For example, "LSV" could indicate "language, sex, and violence," as I see them stamp on TV shows in America (maybe elsewhere, but I only see American TV).

For example, a marriage podcast would most likely have the S rating, and I would be fine with that if I'm looking for a marriage podcast. But if I see a non-marriage content with "S," then I can better know what to expect in the episodes, or avoid if it's something I'm not comfortable with.

The whole "PG" and similar rating system, and even age stuff, seems to be breaking down when movie studios are getting away with profanity in rated-G movies And standards of acceptability for different age ranges will be different (perhaps drastically) for many people.

**Comment by jamescridland** [comment]
Movie ratings are the result of a regulator, who watches the movie and says something like "that's a 15, but you could get it to be a PG if you got rid of two fucks and a shit". Film producers then negotiate and resubmit it. Podcasters do not have a regulator (though that's an interesting business idea).

TV ratings exist in some countries. Those are, I think, self-rated according to a big book of standards and regulations: but policed by media regulators who fine or even remove broadcast licences if the broadcaster puts out the wrong material at the wrong time. Podcasting has no such media regulator.

The current system of "explicit" and "non-explicit" is confusing enough for some podcasters, where forums are full of "I said the word shit twice, do I have to mark it explicit"? Is the intention here to expect them to self-rate their shows?

For television, [every country has its own ratings system](https://en.m.wikipedia.org/wiki/Television_content_rating_system). The same goes for [movie ratings](https://en.m.wikipedia.org/wiki/Motion_picture_content_rating_system). Is the suggestion here that US culture is the only one that matters?

As one example, let's talk about nipples. In Italy, a TV ad for shampoo may feature a blonde woman using the product in the shower, and, while tastefully shot, a nipple may be in vision for part of the time. In the US, a "wardrobe malfunction" which ended up not even showing a nipple but just part of a breast, was one of the biggest media scandals for some years. Which cultural norm takes precedence here?

In Thailand, most Thais find it deeply rude and disrespectful to be rude about the King (to the point where if you stand on a coin, you'll upset many Thais). What happens here - is it worth a warning about being rude about the King of Thailand, or because it's fine to be rude about him in the US, does US culture overrule the Thais?

If Derek says "I just nipping out to bum some fags" in the UK, this is the perfectly acceptable practice of leaving the pub and asking some strangers to give him some cigarettes. Do I need to mark that with some kind of a warning for other cultures for which this means something quite different? Some Glaswegians consider the term 'cunt' as a friendly way to describe their best mate - so is that OK?

As you may be able to tell, I'm not in favour of this tag. By all means have something that marks something as "Explicit", which I would expect to carry swears or content that is not for young ears; and ideally mark something as "Made For Kids" if it is, indeed, made for kids - but I'd reject a more detailed, nuanced ratings system. Quite apart from being naked American cultural colonialism, it's just unworkable.

**Comment by kilobit** [comment]
++ For **not** getting into the business of defining standards, rating schemes or categories.

There are Podcasters and App developers who would use the a content rating - but it should be the Podcaster specifying the scheme and rating.  Possibly multiples e.g. mpaa and regional.

Beyond this, my thought would be to stay out of it.

Great discussion - Thanks!

**Comment by theDanielJLewis** [comment]
I'm fine with something that indicates "explicit" or "kid-friendly." I think there should also be a gray area in the middle, as there is with `<itunes:explicit>`. 

There was a short time that Apple allowed _only_ one way or the other, but before that, and since then, they allow unmarked to display no advisory tag. My perspective of such is that it's similar to a PG-level movie. In other words, maybe some mild profanity, and mild other stuff some might find offensive, but it's neither fully explicit nor fully kid-friendly. I'm all for maintaining something like that, but I think it _needs_ some definitions.

**Comment by jamescridland** [comment]
Agree - we do need some definitions of what "explicit" is likely to contain. Is that clarified by Apple anywhere?

Is there a kid-friendly tag? For some reason I thought there was (but perhaps that's YouTube).

**Comment by jmikedupont2** [comment]
So we haven't really discussed how the different rating systems work that
we can select from , but I would propose that we could have alternative
systems that are defined by a user group. That way you can select between
official rating systems and non-official rating systems and we have some
way of documenting what the different values are. This is something that
you can do using ontologies. It doesn't have to even be that hard.

**Comment by jmikedupont2** [comment]
So to be clear with my last comment we could do something like this 

"Made For Kids" can be a rating itself.

`<podcast:contentRating>For Kids</podcast:contentRating>`
or 
`<podcast:contentRating standard="podcasting2.0 community">Made For Kids</podcast:contentRating>`

or what I think is most explicit, we us a `rating` attribute to mark the 
`<podcast:contentRating rating="https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/community_rating.md#made-for-kids">Made For Kids</podcast:contentRating>`

`<podcast:contentRating rating="https://github.com/jmikedupont2/podcast-namespace/blob/main/docs/rating.md#made-for-kids">Made For Kids</podcast:contentRating>`

Now if we cannot use an url for the entire rating, say it is missing an anchor for the cell, then we could use two parts like
`<podcast:contentRating rating="made-for-kids" reference="https://github.com/jmikedupont2/podcast-namespace/blob/main/docs/rating.md">Made For Kids</podcast:contentRating>`

That should be good enough to define a community standard or have any community define their own standards how they see fit.

Now if you want to reference this fictional nationstates game island state of Charlotte Ryberg, of 19 years old you could say
`<podcast:contentRating rating=19 reference="https://nsindex.net/wiki/Motion_picture_rating_system#Charlotte-Ryberg">Charlotte-Ryberg 19</podcast:contentRating>`


See this nice table <https://en.wikipedia.org/wiki/Motion_picture_content_rating_system> of all different rating systems 

`<podcast:contentRating rating="A" reference="https://en.wikipedia.org/wiki/Motion_picture_content_rating_system#Bulgaria" rating>Bulgaria A</podcast:contentRating>`

**Comment by jamescridland** [comment]
I don't understand the purpose of this, sorry.

We have a "clean" tag, which indicates it's good for kids ears.

We have an "explicit" tag, which means naughty words.

We don't need to reinvent either of those. What are the benefits of adding additional ratings?

**Comment by jmikedupont2** [comment]
> What are the benefits of adding additional ratings?
First of all I am evaluating the namespace and trying to come up with an atom/owl representation of it, not questioning the idea that we need this field at all. I did not propose it but I see its usefulness. 

A simple binary system is yes simple but contains not only cultural bias but also it lacking in extensibility.

What if you want to use the rotten tomoatoes rating system, they have also multiple ratings, so we would need a count and aggregate function?
`<podcast:contentRating  rating="8.40" reference="https://www.rottentomatoes.com" count=192 aggregate=avg >`

If you follow the idea of allowing for users to come up with different ratings via urls and self defined ontologies that would provide the benefit of ratings that are not just kids/grownups but could be anything at all. 

There is an article on this general topic of open vs closed rating systems and evaluation of them,
https://www.aaai.org/Papers/Symposia/Spring/2005/SS-05-03/SS05-03-009.pdf

**Comment by jmikedupont2** [comment]
We can also include the rating or stars from apple for example with a sample date. 

The number one podcast from today could be marked as such.

`<podcast:contentRating rating="1" reference="http://www.itunescharts.net/us/charts/podcasts/" date="12/08/2020">`

**Comment by jmikedupont2** [comment]
https://wilwilliams.reviews/2017/06/23/wils-rating-system-for-podcasts/ here is wils rating system for example, that would be a good candidate. Also we have not even talked about user generated ratings yet. If I have an api key can I submit a rating to a podcast?

**Comment by cio-blubrry** [comment]
The RawVoice namespace has a rating tag that we set both the TV and Movie standardized ratings to. We did this back in 2008/2009 when Blubrry was making podcast apps for the set top boxes like Roku, Samsung, Boxee, etc.. This is what we learned:

At the time 2008/2009 we had each platform telling us which rating system they required for their platform. Most wanted to know if it had one of 2 things:

- Adult Content
- Violence

The reasons were country based. e.g. India does not allow adult content or violence. US for example they just wanted the app to display a warning.

At the time only one service required the TV ratings, 2 required the movie ratings. As confusing as it is, we have a mapping of the two back and forth, meaning if you set a TV rating we can also figure out your movie rating, and visa/versa. the only challenge was the TV stuff for children, at the time we analyzed that there were not enough child based podcasts to make a difference, but today it does make a difference. In other words, if you want to target children, not just state content is safe for children, the TV ratings are necessary.

Now the biggest challenge. The movie ratings are easy, its clear and you can only pick one. TV ratings are much more complicated, you can pick multiple ratings and attributes. Not only is this confusing for developers but if you ever watch TV today its confusing as a parent when you see the parental advisory and there are a million symbols on the TV and it only shows them for 2 seconds. 

With this feedback, to do this right you may want to plan on nested tags within the contentRating tag, with nested tags for one movie and multiped for TV. Also allow for contentRating to be entered multiple times and have an attribute for countries and one that is for all countries if not specified.

Today you can find the tag in all PowerPress sites if the user sets the attribute. We made it optional, and most podcasters ignore the setting sadly.

My suggestion is to make the tag specific and narrow the focus on just the MPAA movie rating and call the tag <podcast:movieRating> or <podcast:mpaaRating>.

**Comment by jmikedupont2** [comment]
So this is a good suggestion and it fits right into what I'm proposing that
we have a generic system for referencing a rating system by URL and we
propose podcast index rating system that's extended by these two items so
we have now made for children, contains violence, contain sexual content as
three flags in our community quote-unquote ontology from podcast index
other people can use their own ontology and this all fits together so we
should be just fine we can extend our community ontology and other people
can propose their own and it should be all very peachy

**Comment by cio-blubrry** [comment]
I ask this often, but can we have a refresher what the "Goal" is with creating a ratings tag today? 

If the goal is to meet government guidelines, you need to get the FCC involved and they will set the rating level for you.

If the goal is to handle it like the MPAA, then the next step is to create an oversite board. Keep in mind this approach each podcast would apply for the rating and then the board would approve it. I don't think that is practical.

If you just want to create a generic list, plan on it being scrutinized at some point by the FCC. I would not go down that road.

If the goal is to have some ratings to display in the app for the lister to decide, we should pick one of the standards that already exists, or keep it simple like Apple did, either it has explicit content or it doesn't.

Once the "Goal" is clearly defined then we march to that goal. I am afraid many of these tags, as cool as they are, are proposed because its meta information we could put in the feed, not necessarily because it achieves a goal that is universally beneficial for all.

jmikedupont2, I believe you proposed this tag, can you state as clearly as possible what the goal is of the tag, where it will be used and why it is beneficial, that will help us then focus on what information should and should not be in it.

**Comment by jmikedupont2** [comment]
I did not propose anything, I just am working on creating an OWL/ATOM
ontology representation of the podcasting namespace and am researching into
existing work being done. If you ask me we should be open to user defined
and user generated ratings of any type. Note that no body is asking me.

**Comment by cio-blubrry** [comment]
We should have the person who proposed this tag reply with the goal(s) first. We may end up creating a tag that does not solve the purpose of what the original person intended.

**Comment by johnchidgey** [comment]
Just looking here: https://en.wikipedia.org/wiki/Television_content_rating_system there's so many different interpretations globally that any system will be problematic. Even attempting to define what is or isn't a "sexual reference" in the content of an episode of a show is a task in itself and highly subjective from a cultural standpoint (per James Cridland's responses above).
The other concern with this kind of classification is that in a TV/Movie context it's reviewed and approved by a third-party body, whereas this would be proposed (initially) by the podcast creator (a-la 'classification pending'). In order for a listener to trust what's in the RSS Feed, there would need to be a way to cross-check any such rating had been validated by an independent third party. Whilst not impossible, it's problematic.

**Comment by douglaskastle** [comment]
I think one needs to look at this backwards a little bit.  Some people, usually of a religious persuasion, but not always, want to have strong ideas hidden from themselves.  I won't get into the rights and wrongs of that, it just is a case and people want to have the ability to do that. I am Irish, for example, we swear a lot more than Americans and still find it weird when movies and TV shows have the swearing removed or bleeped.

An app could have a system where people would decide to have all content masked until it is self categorised as safe for their ears (explicit content).  There is a loose honours system required here that people report correctly, but there might be a 3rd party involved to monitor and report to at a podcast level is people find the content isn't fitting the reporting.  

Bare in mind there are some devious people out there who will violate any rule if they can get you to click, people like this:

[Misspelled children’s Web sites lead to porn](https://www.networkworld.com/article/2288939/misspelled-children-s-web-sites-lead-to-porn.html) (You'll have to trust me when I say it is safe to click that link, but that is the challenge here)

This system should always be an opt-in, not an opt-out.  If one want to have access to the raw, unfiltered internet of ideas, they should. It is not the job of the standard/podcast social to block information, but to at least allow the ability to filter more appropriately.

**Comment by kilobit** [comment]
++ @douglaskastle 

There is a place for censorship.

That place is in the control of the user - in the form of a filter.  

There are plenty of things in this world that, as a user, I do not want to see, hear or experience.  But it is up to me (or the parent of a minor) to choose what that entails.

A filter as a service - tailored to my preferences, sounds valuable in that sense.

The role of a content rating tag in this circumstance is to inform the user controlled filter.  This is the use-case that the content rating tag should be designed to support.

**Comment by theDanielJLewis** [comment]
> Agree - we do need some definitions of what "explicit" is likely to contain. Is that clarified by Apple anywhere?

Explicit ([source](https://help.apple.com/itc/podcasts_connect/#/itcb54353390)): A label issued by media producers that alerts consumers to content that might be inappropriate for children. See also clean.

Clean ([source](https://help.apple.com/itc/podcasts_connect/#/itcb54353390)): A label used to identify a podcast that doesn’t contain explicit content. See also explicit.

**What's the goal?** I think the goal is to let the listener (or guardian) be more informed about the content of podcasts.

I think this is worth taking a step back to answer this question: should appropriateness be for the listener to decide for themselves, or for the podcaster to decide for them.

I think most TV and movie rating systems are built on someone else deciding for you. For example, Pixar and the MPAA decided that profanity is now acceptable in a rated-G movie created for the whole family. But a rating system that indicates the contents (like "LSV" means language, sexual content, and violence), then it's up to the listener to decide whether it's appropriate.

It's not about "hiding" things, it's about preferences.

Something else worth considering is degrees of any particular content. For example, "L" could mean some simple PG-level profanity, but it could also indicate F-bombs in every sentence. So if following the labeling model, you could use capitalization to indicate severity. Thus, lowercase "s" might mean mild sexual references while uppercase "S" indicates explicit sexual content.

**Comment by jamescridland** [comment]
"Pixar and the MPAA decided that profanity is now acceptable in a rated-G movie created for the whole family " - The Incredibles 2 absolutely got a G rating. In America. But in the UK, the BBFC awarded it a PG, rather than a U. (A 'U' is what Americans call a 'G'). Because culture is different across the world.

We have 'Clean', and 'Explicit'. I think we should have a 'Made for Kids' as well. I once listened with my daughter to a podcast that was promoted as suitable for young girls, and my six year-old was told a story about a miscarriage followed by cancer. But, you know, no fucking swears, so that's just fine.

However, I do come back to the central point: given that most podcasters appear not to even understand when to use an 'Explicit' tag, I feel it wholly unsuitable to have a self-regulating and optional rating system. Let Apple deal with this - they have much more at stake in terms of brand reputation. This shouldn't be anything that we want to go near, in my opinion.

**Comment by jmikedupont2** [comment]
This is really the question of are we going to delegate to the
establishment or support the community in defining its own terms.

**Comment by jamescridland** [comment]
Or... a different idea:

```
<podcast:contentAdvisory>
Contains discussions about murder and mild cuss words
</podcast:contentAdvisory>
```

I'd be much happier with this, but it's certainly not programmatically useful.

**Comment by jmikedupont2** [comment]
If we allow user defined urls to define the contents we can defer to the
community and over time it will stabilize. this is the entire idea behind
rdf.

**Comment by theDanielJLewis** [comment]
Whoa. @jamescridland and I just got on the exact same frequency! I was just coming here to suggest we make it a text field, allowing podcasters to enter their own text that they deem appropriate. But maybe we can suggest some standards or even include some of the other ratings standards as attributes.

```xml
<podcast:contentAdvisory mpaa="pg" contents="vl">
Contains discussions about murder and mild cuss words
</podcast:contentAdvisory>
```

The idea here is that the plain language communicates more accurately, while the attributes could allow for searching or filtering.

**Comment by jmikedupont2** [comment]
the reason why a url will be better than free text is because it will be
easier to define the meaning and eventually the community will agree. the
text you want will be defined behind that url

**Comment by douglaskastle** [comment]
@jamescridland 

> Let Apple deal with this - they have much more at stake in terms of brand reputation. This shouldn't be anything that we want to go near, in my opinion.

This seems like a weird point to get hung up on.  The purpose of a standard is to help develop the tools that allow people to build something. Not actually build it.  Putting the feature in and enabling work is supposed to be the goal.  The standard is not filtering, it is allowing the hooks for filtering. Filtering would be done by something else, a podcatcher, website, etc. 

Second, leaving something in Apple's hands is the reason why people are here to help break the wall garden model that is starting to form around podcasts.  You yourself may not want to shoulder the reputational load of managing content ratings (neither would I to be fair) but there may be some innovator out there that can see and do something that we can't.

I agree that self rating is a of limited use.  Most people would like to filter out racist, bigoted etc content.  I doubt many people would self rate themselves on that.  Though this is the internet, nothing surprises me any more.

**Comment by jamescridland** [comment]
[This episode](https://podcasts.apple.com/us/podcast/play-episode/id208516732?i=1000501246795) of "Answer Me This" contains the sort of thing I'd like to see in a content advisory:

> CONTENT WARNING: the episode may not be suitable for children, as it contains contains information about parental deceit w.r.t. Christmas logistics, plus a thoroughly researched passage about Adult Material depicting Jesus.

This podcast is labeled as 'explicit', but I like this detail. And good luck with a URL about "parental deceit" or "adult material depicting Jesus"...

**Comment by jmikedupont2** [comment]
@jamescridland these are good points and if a podcast has unique content then the URL can just point to a page that they control or even reference an anchor on the main site there are Anonymous nodes and many other ways to reference content via a URL we can talk about all of this and you know we don't have to do all of it in version 1

**Comment by daveajones** [comment]
This tag started out as merely interesting.  But, this makes it feel like something that is providing useful information that is new.

**Comment by theDanielJLewis** [comment]
I think it should be a text advisory, not a URL. A podcast app could pull in that text advisory plus use the attributes for filtering or searching. A URL sounds like yet something else to manage per show or even per episode.

@jamescridland's example is great to illustrate where there's not an outright "standard" (and thus no helpful data for the attributes), but the label can help listeners decide whether they find _that_ content acceptable.

With today's highly triggered culture, this kind of thing can significantly help people make better choices about their listening.

It's very similar to the MPAA ratings for movies. While two movies could be rated "R," they both have additional details: one says "Rated R for violence" while the other says "Rated R for sexual content." While I might want to filter out _all_ R-rated movies for children, I might be more particular for myself finding fictional violence acceptable but not sexual content. Thus, I can make a better decision for myself about what movies I watch.

That's the kind of empowerment I want this to give podcast-consumers (or guardians), as well as giving podcasters better abilities to properly inform their audiences.

**Comment by daveajones** [comment]
Agreed.  I think I'll go ahead and put this in the phase 3 list.

**Comment by jmikedupont2** [comment]
I covered this topic on the second half of today's podcast on the stream of
random and I will have a detailed response with examples for my proposal in
the future we still have some time to discuss all of this

**Comment by daveajones** [comment]
Oh yes.  Plenty of time.  This is for a later phase so no need to rush.

**Comment by jamescridland** [comment]
`<podcast:contentAdvisory explicit="yes" madeforkids="no" clean="no">
This episode may not be suitable for children, as it contains contains information about parental deceit w.r.t. Christmas logistics, plus a thoroughly researched passage about Adult Material depicting Jesus</podcast:contentAdvisory>`

Although I do dislike the idea of duplicating the iTunes explicit/clean tags.

**Comment by jamescridland** [comment]
`<podcast:contentAdvisory explicit="no" madeforkids="no" clean="yes">Contains mildly arousing namespace talk</podcast:contentAdvisory>`

**Comment by daveajones** [comment]
> `<podcast:contentAdvisory explicit="no" madeforkids="no" clean="yes">Contains mildly arousing namespace talk</podcast:contentAdvisory>`

LOL.  Now we're talking.

**Comment by theDanielJLewis** [comment]
Is it really necessary to have attributes for `explicit`, `madeforkids`, and `clean`? Is there any case where something would be explicit _and_ made for kids? Or explicit _and_ clean? I think it could be simplified to a single attribute that gets any of these as a value, not as separate attributes.

Then again, I wonder if we should make the labels even more clear altogether, building on the idea of`madeforkids`, we could have `kids`, `general` (the assumption if missing the attribute or the whole advisory tag), and `adults`.

So I would suggest it look more like these.

```xml
<podcast:contentAdvisory audience="kids">Contains mildly amusing namespace talk</podcast:contentAdvisory>

<podcast:contentAdvisory audience="general" contains="sl">Contains mild profanity and sexual references</podcast:contentAdvisory>

<podcast:contentAdvisory audience="adults" contains="S">Contains explicit sexual references</podcast:contentAdvisory>

<podcast:contentAdvisory audience="adults" contains="V">Contains audio depictions of gory namespaces and torture</podcast:contentAdvisory>
```

Then again, might we want to use ages instead? I know of two clean-comedy podcasts where one specifically says, "ages 10 and up" because while it's _clean_, parents might not want their youngest children listening. Or, this could be a way to indicate a specific target, like younger kids, older kids, teens, or even retirement ages.

With that in mind, we could make the tags like this:

```xml
<podcast:contentAdvisory audience="5-8"></podcast:contentAdvisory>

<podcast:contentAdvisory audience="10+">Contains mildly amusing namespace talk</podcast:contentAdvisory>

<podcast:contentAdvisory audience="general" contains="sl">Contains mild profanity and sexual references</podcast:contentAdvisory>

<podcast:contentAdvisory audience="18+" contains="S">Contains explicit sexual references</podcast:contentAdvisory>

<podcast:contentAdvisory audience="18+" contains="V">Contains audio depictions of gory namespaces and torture</podcast:contentAdvisory>

<podcast:contentAdvisory audience="55+" contains="l">Contains mild profanity and discussions about your nearing death</podcast:contentAdvisory>

<podcast:contentAdvisory audience="21+" contains="A">Contains 100-proof ethanol</podcast:contentAdvisory>
```

Of course, when you get into things like alcohol, that's also different by countries and even by some states/provinces. But the whole idea is to advise, "We think this is appropriate for this group with these caveats or reasons. Make your own decision."

**Comment by jamescridland** [comment]
I'm liking "audience" being simply an age range, as in your second example. I think that's understandable by podcasters. I don't know the LSVA things you're linking to, but if that's a relative standard, I think this is rather better than anything more cultural.

"Kids" is not that helpful -  4 year-old has very different content requirements than a 9 year-old, after all.

Alcohol is obviously a 16+, 18+ or 21+ thing. But I think putting the onus on the podcaster is the right thing here, as long as it's really obvious how to rate things.

**Comment by theDanielJLewis** [comment]
LSVA and such are simply the [American FCC's abbreviations for content](https://en.wikipedia.org/wiki/TV_Parental_Guidelines#Content_descriptors). L for language, S for sex, V for violence, also maybe add A for alcohol, D for drugs, and probably some others. In my examples, I use uppercase to indicate excessive or explicit levels of each. For example, Marc Maron's show would have "L" because of its heavy profanity, but an NPR show might have "l" (lowercase) because of some mild profanity.

**Comment by theDanielJLewis** [comment]
I also found [these advisory labels](https://en.wikipedia.org/wiki/United_States_pay_television_content_advisory_system#Advisory_labels) that are more granular than capitalization.

For additional reference, [look at how IMDB segments their detailed advisories](https://www.imdb.com/title/tt2948372/parentalguide?ref_=tt_stry_pg).

**Comment by theDanielJLewis** [comment]
I just noticed that Libsyn offers these options for individual episodes:

<img width="384" alt="Screen Shot 2021-01-13 at 10 00 27 AM" src="https://user-images.githubusercontent.com/154936/104469182-45cd6780-5586-11eb-95ea-35e4d3f19e37.png">

**Comment by theDanielJLewis** [comment]
I just learned that those TV rating tags don't add anything to the Libsyn feed. Thus, I suspect they're removed in the upcoming "Libsyn Five."

Nonetheless, it's an example (along with PowerPress's) of how particular contents could be (and have been) disclosed.

**Comment by daveajones** [comment]
What if we remove all attempts to “rate” the content. And instead just focus on the “advisory” aspect.  The usage of the tag can be to “advise the listener _in advance_ to any aspect of your content that may not be for a general audience”.  

Obviously nothing can capture 100% of this intent.  But, this could be thought of as a “trigger warning” so to speak.  Just a short snippet that podcast apps are meant to highlight in the episode details, or even as a modal pop up before allowing the episode to play (“Content Advisory: This episode contains a discussion of school bullying that may be emotionally difficult for certain audiences.” Play/Cancel).  That type of content is clearly of interest to kids. But, giving a warning like that allows them to choose if they are emotionally ready to hear this content at that moment. It allows them to make a more informed decision without treating them as if they can’t handle it.

So, instead of any attempt at classifiers the tag could just be:

`<podcast:contentAdvisory warn=“true”> This episode contains a discussion of school bullying that may be emotionally difficult for certain audiences.<podcast:contentAdvisory>`

**Comment by daveajones** [comment]
Another version might be:

<podcast:contentAdvisory warn=“False”> This episode contains discussions of religion and theological topics - primarily the differences between Islam and Judaism.<podcast:contentAdvisory>

Something like that doesn’t need an interstitial warning inserted before playing. But it does help to let the listener know what they are about to hear. Maybe they just wanted to listen to history of religion, and have no desire to listen to a theological debate.

**Comment by theDanielJLewis** [comment]
I don't think there should be any interstitials, only a visible advisory.

But I still think disclosing the contents would be helpful to developers, podcasters, and consumers.

Also, it could help reduce having to see any advisories. For example, if a _podcast_ is marked as containing "LSV," then any episode that has one or all of those three content types might not need to display the advisory since the consumer was aware of the advisory when they subscribed in the first place.

**Comment by theDanielJLewis** [comment]
Hearing that TikTok will start allowing content to be rated for age-appropriateness, I'd like to bump this idea again.

I think things could come down to two approaches:

1. Age-based advisories, like "all," "13+," "18+." However, even these things don't really indicate that _kind_ of potentially objectionable content. For example, 18+ might be simply because it discusses alcohol (which is actually 21+ in nearly all the USA), not because it's pornographic content. Thus, I still think the other approach is better.
2. Content-based advisories, following a simple word list or letter code apps can interpret and then display appropriate advisory tags. For example, "sex," "violence," "language," "drugs," or "alcohol" (full words or first letter of each).

There could, however, be a combination of these. Like the content indicators _and_ a suggested minimum age.

I'm still a bigger fan of #2, because it communicates more information that can help make the appropriate decisions.

**Comment by jamescridland** [comment]
I am **vehemently** against Americans dictating their culture to the rest of us. Minimum age ratings are simply not appropriate in all cultures. Nor is any US-based cultural rating like TV or movie ratings which rightfully differ to those in other countries. They are also, as Daniel points out, utterly useless for the use-case of "will I find this content profoundly upsetting". A better plan is to explain WHAT the content is, and potentially WHY you might find it upsetting.

As above, adding tags for `explicit` (a mirror of iTunes) and `made-for-kids` (content made for children) could be helpful. "Explicit" isn't very clear nor obvious, but Apple has a similar tag and it's typically used for content of a sexual nature or for heavy swearing. The made for children tag is one that YouTube has produced and it seems clear in context, though there's a definition of what "children" means here.

A "content advisory" plain text comment could be parsed for keywords; but is just that - an advisory of the topic from the creator should they seem it necessary. If you like, it's an equivalent of a "trigger warning" that you might see elsewhere. It is what the _creator_ believes someone should know about an episode.

Some examples (most of which, I don't think, would deserve the explicit tag):

`<podcast:contentAdvisory explicit=“false” madeforkids="false">Indigenous listeners are warned that this episode contains the voices of people who are deceased.<podcast:contentAdvisory>`

... this is a good example, since indigenous Australians believe it is extremely bad luck to hear or see any representation of dead people, and actively wish to avoid this. This warning appears in front of many archive shows in Australia. How do we get this over with a US-based television ratings system, or something devised to shield people from pictures of Janet Jackson's tits?

(Interesting fact: 'tits' is a banned word on FCC-licensed radio stations).

`<podcast:contentAdvisory explicit=“false” madeforkids="false">This episode discusses recreational drug use.<podcast:contentAdvisory>`

... different recreational drugs are, of course, illegal in different parts of the world - including alcohol.

`<podcast:contentAdvisory explicit=“false” madeforkids="false">This episode contains a description of a suicide.<podcast:contentAdvisory>`

`<podcast:contentAdvisory explicit=“false” madeforkids="false">This episode has an interview with a rape victim which some listeners may find disturbing. The episode discusses reproductive rights.<podcast:contentAdvisory>`

...one person's sensible discussion about reproductive rights is, after all, another person's "barbarism", or another person's talk about "murdering babies".

`<podcast:contentAdvisory explicit=“true” madeforkids="false">This episode contains a graphic commentary of a school shooting.<podcast:contentAdvisory>`

`<podcast:contentAdvisory explicit=“false” madeforkids="false" />`

I would advise around this tag that if there is text in this field, **it should be visible before playing.**

As to whether it appears in the **channel** rather than the **item** - I'm unsure about that; but since the `explicit` tag does so, perhaps it's easiest to suggest that it can appear there, too; in which case, text should be visible before subscribing/following.

**Comment by theDanielJLewis** [comment]
> As above, adding tags for explicit (a mirror of iTunes) and made-for-kids (content made for children) could be helpful. "Explicit" isn't very clear nor obvious, but Apple has a similar tag and it's typically used for content of a sexual nature or for heavy swearing. The made for children tag is one that YouTube has produced and it seems clear in context, though there's a definition of what "children" means here.

Apple defines the clean tag as content appropriate for children, and explicit as inappropriate for children.

I agree with @jamescridland's suggestion on a plain-text field. I would consider that a more clear advisory than the category system, but requiring custom text would probably cause the podcaster to ignore it. That's why I think a category system, as a baseline, could be easiest. The publishing tool could ask, "What content advisories should be included with this show/episode?" and then list the standard set.

And listening to what James is communicating, I think I see something that could be clarified: I'm suggesting this tag for indicating content people will want to avoid or make other decisions on, not just to warn that something could be triggering.

For example, a liberal might be triggered or upset by conservative viewpoints, and a conservative might be triggered or upset by liberal viewpoints. So I'm not suggesting an advisory for that kind of thing, but for content that is actually damaging in some way. For example, discussions about alcohol or drugs could lead an addict to relapse, sexual content (either pornographic or other) could be inappropriate for children.

I'll use myself as an example. I don't cuss and I don't like hearing foul language. But when I do hear it, it doesn't cause me damage; it's merely something I don't prefer. I might still listen/watch as long as the profanity isn't excessive. But if I knew ahead of time that a podcast had profanity, I definitely wouldn't play it out loud around my son.

But I don't expect any kind of "trigger warning" for a podcast with politics or religion different from my own. I think that kind of thing is excessive.

@jamescridland, from your qualified observations, do non-American English speakers seem pretty aware of what's considered profanity to Americans? From my own observation, Americans seem to be more ignorant about non-American English. (Americans can't even stick with American English in some spellings and pronunciations! But I'm guessing non-Americans don't make the same mistakes. For example, I don't think I've ever heard a British person pronounce "niche" the American way, but I hear Americans pronounce it inconsistently.)

**Comment by jamescridland** [comment]
> Apple defines the clean tag as content appropriate for children, and explicit as inappropriate for children.

So, perhaps we can assume that `explicit="clean"` is the equivalent of YouTube's Made For Kids?

I am really against a category system, since it is necessarily US-centric. How does it work in the case of the indigenous warning that an episode contains deceased people? Or here's another:

`<podcast:contentAdvisory explicit=“clean">This episode criticises the Thai monarchy.<podcast:contentAdvisory>`

Most Thai people find a discussion criticising the Thai monarchy deeply insulting. Are you proposing a category system that includes criticism of the monarchy?

> I'm suggesting this tag for indicating content people will want to avoid or make other decisions on, not just to warn that something could be triggering.

I don't understand the delineation here. "This podcast contains a lot of swearing" is the type of description that you might want to see. Few creators seek to deliberately upset.

> do non-American English speakers seem pretty aware of what's considered profanity to Americans?

[This gloriously profanity-filled piece of research by the UK regulator](https://www.ofcom.org.uk/news-centre/2021/why-offensive-language-research-matters) contains a full discussion of what British people find offensive. I'm prepared to bet that it differs quite a lot from the US. Here's a bit of the research about "names of parts of the body":

![Screen Shot 2022-07-21 at 2 40 05 pm](https://user-images.githubusercontent.com/231941/180131083-37736921-2030-413a-aa47-5f415b5b887e.png)

Meanwhile, in Australia, the word "cunt" is, in many circles, a term of affection between mates. "Fuck me, it's Bruce! Come over here, cunt! Get this tinnie down yer, mate!" - even if it's one of the strongest words you can use in the UK. It could well be heard on the TV here, though I'd doubt it would necessarily be editorially justified.

The current top 40 hit by Gayle, "ABCDEForgetU", will be being played on US commercial radio; but the explicit version, "ABCDEFuckU", would be perfectly acceptable on Australian radio at any time of the day - if (and only if) the station gave a language warning first. The format of the language warning? Someone saying: "LANGUAGE WARNING!" suffices. Imagine my horror, having grown up in the UK where such language would never make it to the radio, to hear this blasting out of the car radio at 12.30pm!!

All this - partially gratuitous - swearing is really only to underline my considered opinion that *anything* further than a simple explicit/clean tag adds US cultural norms onto this system, and this is to be resisted.

- - -

I wonder if the "right" way to do a content advisory is merely to use explicit/clean, and to advice all podcasters to make any specific content advisory within the audio? That would be fool-proof: much more so than a plain-text field.

**Comment by theDanielJLewis** [comment]
> So, perhaps we can assume that `explicit="clean"` is the equivalent of YouTube's Made For Kids?

I see those things as different. "Made for Kids" has a straightforward meaning. If I see something is made for kids, then I'm going to think it's made for kids (profound, I know). In other words, that kids are the intended audience. But when I see the "clean" tag, I just know the show is free from profanity, sexual content, and other things, not that it's made for a particular age group.

Maybe my idea of a rating system is DOA in today's loud society.

**Comment by theDanielJLewis** [comment]
Put another way, I'm looking for a way people can anticipate the morality of content before they consume it. That's essentially what the entertainment ratings do.

**Comment by tomrossi7** [comment]
This has been a long discussion, but we do have a good reason to land somewhere. YouTube has asked that we help them identify podcasts that are "made for kids". I believe this is a result of[ COPPA](https://www.ftc.gov/legal-library/browse/rules/childrens-online-privacy-protection-rule-coppa) laws. I wonder if there is a tag we could land this plane in a simple implementation that accomplishes at least identifying podcasts that are made for kids?

**Comment by theDanielJLewis** [comment]
In other words, categorizing content by what's:

- _made for_ kids
- _allowed_ for kids (without being made specifically for them)
- _restricted_ for kids (made for adults only—which would include obscene content such as _Call Her Daddy_ as well as mature content like a Christian podcast about sex within marriage)

**Comment by tomrossi7** [comment]
It doesn't seem like we are converging on something that could be adopted as a standard for degrees of explicit/clean. What if we just do the absolute simplest:

`<podcast:madeForChildren>true</podcast:madeForChildren>`

or if we want to take the explicit tag out of the iTunes namespeace, we could do something like this:

`<podcast:explicit madeForChildren="true">false</podcast:explicit>`

**Comment by theDanielJLewis** [comment]
> It doesn't seem like we are converging on something that could be adopted

Unfortunately, this has largely been forgotten, and most of the engagement has been opposition from only one person.

I've made my case for why we need this and how I think it will help audiences. I just want something more helpful to audiences than Apple's vague and misunderstood "explicit" tag.

**Comment by keunes** [comment]
> I am really against a category system, since it is necessarily US-centric

If we force a specific system, then yes. But **can we think of a way where we support multiple systems?** E.g. in 5 countries the Dutch 'invention' [Kijkwijzer](https://nicam.nl/en/project/kijkwijzer) is used, it'd be helpful if these structures tags could be embedded.

On top, having a flexible system allows for catering the many different moral and social codes that exist throughout the world, as @jamescridland importantly pointed out. Which in some cases might even be required to follow by law.

> On the flipside, Call Her Daddy or a "let's get drunk and talk about our favorite beers" podcast would have a minimum age of 21 with no necessary recommended/target age.

You mean 18, right?
/s (Just to show that even 'simple' age-based 'suitability' labels can't be globally defined)

Similarly, sexual education content in Denmark may be deemed perfect for 4-5 yo, the same content in NL may be deemed more suitable for 6-7 yo. And for no group in the US because it's too much sexuality for up to 8 years, but using too childish phrasing for anyone 9 and up.

> categorizing content by what's:
> made for kids
> allowed for kids
> restricted for kids

Maybe the problem with the discussion here is that it combines/mixes these points. 'Intended for' and 'acceptable for' are/can be quite different. Which is also why 'Made for kids' ≠ 'Not explicit', as @theDanielJLewis noted.

@jamescridland already gave concrete suggestions for a text-based description which also supports these two classifications:

`<podcast:contentAdvisory explicit=“false” madeforkids="false">This episode discusses recreational drug use.<podcast:contentAdvisory>`

For me this makes sense. But maybe to unblock this discussion we should split off the 'made for' into a different tag & discussion. (Even though I strongly opposed to this in #462.) Then here we can focus strictly on content warnings.

**Comment by daveajones** [comment]
> > It doesn't seem like we are converging on something that could be adopted
> 
> Unfortunately, this has largely been forgotten, and most of the engagement has been opposition from only one person.
> 
> I've made my case for why we need this and how I think it will help audiences. I just want something more helpful to audiences than Apple's vague and misunderstood "explicit" tag.

Not forgotten by me. Just needs lots of time to bake and be discussed. It’s a big one and have to get it right. Plan to discuss on the board meeting tomorrow.

**Comment by daveajones** [comment]
What if there was a country code attribute to specify the jurisdiction that is assumed by what the tag is expressing.  That makes ‘madeForChildren’ make more sense to the app.

**Comment by theDanielJLewis** [comment]
Indicating the originating country could be helpful. I’m most interested in indicating the kind of content and then leaving it to the audience or parent discretion. For example, anything marked with sexual content would be hidden from kids.

**Comment by jamescridland** [comment]
> Indicating the originating country could be helpful.

I would argue emphatically that no, it wouldn't.

This tag is designed to inform people about the content in a podcast.

Am I supposed to read up on the cultural norms of the originating country, then make an informed decision on whether the cultural norm of the country that this came from is going to be appropriate to the country where I am in?

As one example - a podcast with a picture of Michelangelo's "David" is apparently pornographic in some US states. Without knowing the minutiae of US cultural norms, I don't know whether that has come from Florida, Texas or Massachusetts. It sure doesn't help me as a parent work out whether it's an appropriate picture for a child of 8. (Clue: it is).

Given the impossibility of even getting podcasters to understand what "explicit" means, I would *strongly* advise that this is kept as simple as possible.

```
<podcast:contentAdvisory explicit=“false” madeforkids="false">
   This episode discusses recreational drug use.
</podcast:contentAdvisory>
```

This seems good. Two computer-readable tags. And a content advisory with any nuance that can be surfaced in UX.

Please resist the temptation to do anything more with this.

**Comment by keunes** [comment]
> Am I supposed to read up on the cultural norms of the originating country, then make an informed decision on whether the cultural norm of the country that this came from is going to be appropriate to the country where I am in?

No. Should I assume US/western cultural norms because that's the folks that decided on the podcast namespace classification?

I think leveraging localised systems is the least worst alternative to the US dictating the world _if_ we want some structured content _rating_.

We can decide to just go the **contentAdvisory** way. But I would argue that's a different thing and can (and should) co-exist next to **contentRating**.

If US users don't understand Kijkwijzer which must be applied to podcasts from our public broadcaster. Well, tant pis – too bad for them. If so important, apps or platforms utilising this structured data can 'translate' it (e.g. 18 in NL ≈ 21 in US).

At the same time, 'country' is probably the defining characteristic for _ratings_, it should probably rather be name of the content rating system or body. E.g. [ACB](https://en.m.wikipedia.org/wiki/Television_content_rating_system#Australia) and [NICAM](https://en.m.wikipedia.org/wiki/Television_content_rating_system#Netherlands).

And I agree that _advisories_ don't need a system/country classification. They should be human readable/understandable.

```
<podcast:contentRating system="NICAM" rating="14;fear" />
<podcast:contentAdvisory explicit=“false”>This episode contains the voices of people who are deceased.<podcast:contentAdvisory>
```

**Comment by keunes** [comment]
> Am I supposed to read up on the cultural norms of the originating country, then make an informed decision on whether the cultural norm of the country that this came from is going to be appropriate to the country where I am in?

No. Should I assume US/western cultural norms because that's the folks that decided on the podcast namespace classification?

I think leveraging localised systems is the least worst alternative to the US dictating the world _if_ we want some structured content _rating_.

We can decide to just go the **contentAdvisory** way. But I would argue that's a different thing and can (and should) co-exist next to **contentRating**.

If US users don't understand Kijkwijzer which must be applied to podcasts from our public broadcaster. Well, tant pis – too bad for them. If so important, apps or platforms utilising this structured data can 'translate' it (e.g. 18 in NL ≈ 21 in US).

At the same time, 'country' is probably the defining characteristic for _ratings_, it should probably rather be name of the content rating system or body. E.g. [ACB](https://en.m.wikipedia.org/wiki/Television_content_rating_system#Australia) and [NICAM](https://en.m.wikipedia.org/wiki/Television_content_rating_system#Netherlands).

And I agree that _advisories_ don't need a system/country classification. They should be human readable/understandable.

```
<podcast:contentRating system="NICAM" rating="14;fear" />
<podcast:contentAdvisory explicit=“false”>This episode contains the voices of people who are deceased.<podcast:contentAdvisory>
```

**Comment by Marzal** [comment]
Hi, as per https://github.com/Podcastindex-org/podcast-namespace/discussions/749 and https://github.com/Podcastindex-org/podcast-namespace/discussions/695 we are closing old issues.

Please open a new [Discussions](https://github.com/Podcastindex-org/podcast-namespace/discussions) if you think that the matter it's worth pursuing.

Related: https://github.com/Podcastindex-org/podcast-namespace/discussions/454
